cookbooks/travis_postgresql/attributes/default.rb

```diff
@@ -1,5 +1,5 @@
 default['travis_postgresql']['default_version'] = '9.2'
-default['travis_postgresql']['alternate_versions'] = %w[9.3 9.4 9.5 9.6]
+default['travis_postgresql']['alternate_versions'] = %w[9.3 9.4 9.5 9.6 10]
```

Comment:
Can we shift these so that `9.3` is the default and the alternates start at `9.4`?